### ISSUE TITLE
When installing the Flutter 3.19.3 version IPA on a Mac computer with an M1 Pro chip, the related Flutter pages show a white screen, but the click events can still be responded to normally.

### ISSUE THREAD

**Issue by Allenzhan1225** [flutter/flutter#159125]
### Steps to reproduce

1、In our testing, with our Flutter version 3.19.3, when installing our IPA on a Mac with an M1 Pro chip running macOS 12.3, the pages display a white screen. However, click events are responding correctly, and page navigation functions normally.

2、However, our project, using Flutter version 2.10.x, displays correctly when the IPA is installed on a Mac with an M1 Pro chip running macOS 12.3.

### Expected results

Flutter 3.19.3 does not have explicit system restrictions that would prevent it from running on macOS 12.3 with an M1 Pro chip. The occurrence of a white screen while still being able to respond to events could be due to compatibility issues or bugs specific to this version of Flutter when running on this particular configuration. It might be worth investigating potential changes or updates in this version of Flutter that could be causing this behavior.

### Actual results

The screen is white, but the events can still be triggered, and page navigation works normally.

### Code sample

<details open><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[Paste your output here]
```

</details>

**Comment by Allenzhan1225** [comment]
[https://youtu.be/NEkRAtVTuqU](url)

**Comment by darshankawar** [comment]
@Allenzhan1225 
In order to properly address the issue, we would need a minimal reproducible code sample (without third party implementation). I have macOS 12.2.1 so I can verify this behavior if you could provide us runnable code sample.

Also, you mentioned 3.19.3, which is not latest. Current is 3.24.5, so I suggest you to upgrade to latest and re-run your scenario to check if it still exist or not. Even if it is considered to be a bug, the fix won't be applied / backtracked to 3.19.

**Comment by Allenzhan1225** [comment]
@darshankawar Currently, I have created a demo using Flutter version 3.24.5 with flutter create. When running the demo on a Mac running macOS 14.6 and selecting "My Mac (Designed for iPad)", it displays and responds to clicks normally. Additionally, the IPA generated from the archive installs and functions correctly on this machine. However, when this IPA is installed on another Mac running macOS 12.3, it shows a white screen. This is demonstrated in the video below. https://www.youtube.com/shorts/igSmjaL6m1Q

**Comment by Allenzhan1225** [comment]
Here is a minimal reproducible code sample. I created a project using flutter create xxDemo. The code is as follows:



import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
        useMaterial3: true,
      ),
      home: const MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({super.key, required this.title});
  final String title;
  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  int _counter = 0;

  void _incrementCounter() {
    setState(() {
      _counter++;
    });
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
        title: Text(widget.title),
      ),
      body: Center(
 
        child: Column(
   
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            const Text(
              'You have pushed the button this many times:',
            ),
            Text(
              '$_counter',
              style: Theme.of(context).textTheme.headlineMedium,
            ),
          ],
        ),
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: _incrementCounter,
        tooltip: 'Increment',
        child: const Icon(Icons.add),
      ),
    );
  }
}

**Comment by darshankawar** [comment]
@Allenzhan1225 I tried to create an ipa file but it doesn't support on my mac machine config. I also tried to generate from latest mac machine to run on my machine, but it isn't supported. In your case, if you generate the ipa from macOS 12.3 itself using above code (flutter build ipa), does it run or still show white screen ?


> There is an adaptation requirement in the project.

See if [this](https://docs.flutter.dev/platform-integration/macos/building#building-macos-apps) helps.
